syncserver/src/tokenserver/extractors.rs

```diff
@@ -455,9 +451,12 @@ impl FromRequest for AuthData {
 
             match token {
                 Token::BrowserIdAssertion(assertion) => {
-                    let mut tags = Tags::default();
-                    tags.tags
-                        .insert("token_type".to_owned(), "BrowserID".to_owned());
+                    // Add a tag to the request extensions
+                    req.add_tag("token_type".to_owned(), "BrowserID".to_owned());
+
+                    // Start a timer with the same tag
+                    let mut tags = HashMap::default();
+                    tags.insert("token_type".to_owned(), "BrowserID".to_owned());
```

Comment:
duplicating the tag feels a bit awkward, but not quite sure how to resolve it. maybe add an `extend()` in a future PR?